github/pulls_reviewers_test.go

```diff
@@ -58,12 +58,12 @@ func TestListReviewers(t *testing.T) {
 	mux.HandleFunc("/repos/o/r/pulls/1/requested_reviewers", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
 		testHeader(t, r, "Accept", mediaTypePullRequestReviewsPreview)
-		fmt.Fprint(w, `[{"login": "octocat","id": 1 }]`)
+		fmt.Fprint(w, `[{"login": "octocat","id":1}]`)
```

Comment:
Is there a reason you decided to keep the space between `"login":` and `"octocat"`, but not between `"id":` and `1`?